src/block/block_ext.c

```diff
@@ -1343,11 +1344,12 @@ __block_extlist_dump(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_EXTLIST *el,
 
     WT_ERR(__wt_scr_alloc(session, 0, &t1));
     if (block->verify_layout)
-        WT_ERR(__wt_msg(session, "%s extent list %s, %" PRIu32 " entries, %s bytes", tag, el->name,
-          el->entries, __wt_buf_set_size(session, el->bytes, true, t1)));
+        level = WT_VERBOSE_NOTICE;
     else
-        __wt_verbose(session, WT_VERB_BLOCK, "%s extent list %s, %" PRIu32 " entries, %s bytes",
-          tag, el->name, el->entries, __wt_buf_set_size(session, el->bytes, true, t1));
+        level = WT_VERBOSE_DEBUG;
```

Comment:
Should we retrieve the verbosity level associated to the category `WT_VERB_BLOCK` here?

Reply:
I think if we're not using `verify_layout`, we want the message to be only displayed at the `DEBUG` level. Otherwise retrieving the verbosity level associated with the category `WT_VERB_BLOCK` would mean the message always prints.